src/Fantomas.Tests/CommentTests.fs

```diff
@@ -1111,3 +1111,88 @@ namespace Foo
 module Bar =
     let Baz () = ()
 """
+
+[<Test>]
+let ``starting of a multi-line comment should not be removed (with Begin of Comment), 1223`` () =
+    formatSourceString
+        false
+        """
+module Foo =
+    let v1 = method7
+            (* Begin of Comment
+       Body of Comment
+             *)
+    printf "Hello World!"
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+module Foo =
+    let v1 = method7
+    (* Begin of Comment
+       Body of Comment
+             *)
+    printf "Hello World!"
+"""
+
+[<Test>]
+let ``starting of a multi-line comment should not be removed (withOut Begin of Comment), 1223`` () =
+    formatSourceString
+        false
+        """
+module Foo =
+    let v1 = method7
+            (*
+       Body of Comment
+*)
+    printf "Hello World!"
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+module Foo =
+    let v1 = method7
+(*
+       Body of Comment
+*)
+    printf "Hello World!"
+"""
+
+[<Test>]
+let ``starting of a multi-line comment should not be removed`` () =
+    formatSourceString
+        false
+        """
+module Foo =
+    let private GetConfirmedEtherBalanceInternal (web3: Web3) (publicAddress: string): Async<HexBigInteger> =
+        async {
+             let! blockForConfirmationReference = GetBlockToCheckForConfirmedBalance web3
+(*
+             if (Config.DebugLog) then
+                 Infrastructure.LogError (SPrintF2 "Last block number and last confirmed block number: %s: %s"
+                                                  (latestBlock.Value.ToString()) (blockForConfirmationReference.BlockNumber.Value.ToString()))
+ *)
+            return blockForConfirmationReference
+        }
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+module Foo =
+    let private GetConfirmedEtherBalanceInternal (web3: Web3) (publicAddress: string): Async<HexBigInteger> =
+        async {
+            let! blockForConfirmationReference = GetBlockToCheckForConfirmedBalance web3
+(*
+             if (Config.DebugLog) then
+                 Infrastructure.LogError (SPrintF2 "Last block number and last confirmed block number: %s: %s"
+                                                  (latestBlock.Value.ToString()) (blockForConfirmationReference.BlockNumber.Value.ToString()))
+ *)
```

Comment:
The trivia is assigned to [SynExpr_YieldOrReturn](https://fsprojects.github.io/fantomas-tools/#/trivia?data=N4KABGBEDGD2AmBTSAuKBbBBXANosAYrLGALwA6ATuQHYQR4AuYADpQJYBuAho-gOKJGAYVg0AZu0rpE8AKKMAFokoAhbjm41oiAJI0%2BlGhrAAKAO6IARgGY0AdWs2AlGZZYrOdtACC8eJSIAM5BaEGMHDQA5s5oPkEAntoAPAASiAAequxR%2BnxRKgB8ZFS09OVg3InaYMCldBWNYEwAhGCesNAA1kSUohJS6LzsYgBKiOIqiNr4pGCCjKo4nV0AKrDCyt29-ZLSsuqaM2CWtvWmAFT1TY3s4ma7OQB0ACLWWFEAMrAxYErT1xuN304koVQiWGgjCwgSe3yickolFglDMAGUAAqRRgEABMYHIkE%2B4Pay26YBoWHQVhUlRo8GaJLgA32DI65Mp1JUaAApKEwHzCYCgSLRWLGqZNHxwksVk8AGoaLCIJ7rNERdjRUzOVymdk9FGPaTDMYTKYzJ6y7oAOSpNMoCqVKrVGq1Ouc1wuHuoDRFgWhRlJKx2Yj2Q0YIxo40mgRmwoqAF9IAAacBQJCSGjBVBQFNpyDsIIEILsHPiDRBRAgBNAA) node.

It does not make any sense that it starts at the beginning of the file.
I miss a real reason why it the result did what it did.

If you had:
```fsharp
module Foo =
    let private GetConfirmedEtherBalanceInternal (web3: Web3) (publicAddress: string): Async<HexBigInteger> =
        async {
             let! blockForConfirmationReference = GetBlockToCheckForConfirmedBalance web3
    (*
             if (Config.DebugLog) then
                 Infrastructure.LogError (SPrintF2 "Last block number and last confirmed block number: %s: %s"
                                                  (latestBlock.Value.ToString()) (blockForConfirmationReference.BlockNumber.Value.ToString()))
     *)
            return blockForConfirmationReference
        }
```
as source I believe the result would be the same for no real reason.
This is wide open to interpretation that it should start at the beginning of the file.

Reply:
> This is wide open to interpretation that it should start at the beginning of the file.

Hey @nojaf, 1st I guess you meant "line", not "file". Second, maybe Humberto missed the point of #1223 but my expected behaviour for fantomas is not to make it start multi-line comments at the beginning of the line, but to leave comments where they were originally in the source, and not move them.

Reply:
Hey, yes I meant line indeed.